client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Card } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function UltimateTrafficGuide() {
+	const translate = useTranslate();
+	const cost = 17;
+	const currency = 'USD';
+	return (
+		<Card className="ultimate-traffic-guide">
+			<h1>{ translate( 'Introducing the WordPress.com Ultimate Traffic Guide' ) }</h1>
+			<p>
+				<h2>{ translate( 'Discover today’s most important traffic secrets' ) }</h2>
+			</p>
+			<p>
+				{ translate(
+					'We developed this 96 page guide to teach you every modern website traffic trick you need to know in 2020 and beyond.'
+				) }
+			</p>
+			<p>
+				{ translate(
+					'Here are just a few of the topics we cover in {{i}}The Ultimate Traffic Guide:{{/i}}',
+					{
+						components: {
+							i: <i />,
+						},
+					}
+				) }
+			</p>
+			<p>
+				<strong>{ translate( 'The complete SEO checklist for your website (page 53)' ) }</strong>
+				<br></br>
+				{ translate(
+					'Following this simple 7-step checklist will help you get your site featured at the top of search engine results for your most important keywords (and help you avoid the most common pitfalls that you absolutely must avoid if you want to rank well).'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'Why email is STILL a must-have marketing channel (hint: it’s simple, free, and effective) (page 88)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'This guide will teach you everything you need to know: from which email marketing tools you should use, to how to collect email addresses and write great subject lines, and more.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate( 'Harness the power of social media to expand your reach (page 65)' ) }
+				</strong>
+				<br></br>
+				{ translate(
+					'Discover the best ways to leverage social media channels like FaceBook, Twitter, and LinkedIn to connect with your audience and turn them into raving fans.'
+				) }
+			</p>
+			<p>
+				<strong>
+					{ translate(
+						'How to build a simple “traffic magnet” to instantly attract the right audience (page 7)'
+					) }
+				</strong>
+				<br></br>
+				{ translate(
+					'It’s so effective you don’t even have to come close to a “perfect” site before you launch… even the ugliest site can drive an endless stream of business if you get this right!'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'How to build a simple <b>“traffic magnet”</b> to instantly attract the right audience to your site and convert them to raving fans (It’s so effective you don’t even have to come close to a “perfect” site before you launch... even the ugliest site can drive an endless stream of business if you get this right!)' )` *ES Score: 8*